libs/client/src/group.rs

```diff
@@ -56,4 +56,16 @@ impl KanidmClient {
         )
         .await
     }
+
+    pub async fn group_account_policy_webauthn_attestation_set(
+        &self,
+        id: &str,
+        att_ca_list: &str,
+    ) -> Result<(), ClientError> {
+        self.perform_put_request(
+            &format!("/v1/group/{}/_attr/webauthn_attestation_ca_list", id),
+            vec![att_ca_list.to_string()],
+        )
+        .await
+    }
```

Comment:
Something that came up from someone's suggestions in another library, if we use `impl ToString` instead of `&str` it could be a little easier for everyone from a flexibility perspective, just thinking about future users etc?

```suggestion
    pub async fn group_account_policy_webauthn_attestation_set(
        &self,
        id: impl ToString,
        att_ca_list: impl ToString,
    ) -> Result<(), ClientError> {
        self.perform_put_request(
            &format!(
                "/v1/group/{}/_attr/webauthn_attestation_ca_list",
                id.to_string()
            ),
            vec![att_ca_list.to_string()],
        )
        .await
    }
```

Reply:
Yeah, this one was more about the changes to your new function while we were here 😄